analyses/molecular-subtyping-EPN/01-make_notebook_RNAandDNA.py

```diff
@@ -42,16 +42,10 @@ def group_disease(primary_site):
 # Filtering for ependymoma samples 
 EP = pbta_histologies[pbta_histologies["pathology_diagnosis"]=="Ependymoma"]
 EP_rnaseq_samples = EP[EP["experimental_strategy"] == "RNA-Seq"][["Kids_First_Biospecimen_ID", "primary_site", 
```

Comment:
v. minor comment: can we make the order of columns here and for `WGS_dneseqsamples` the same? Took me longer than I would have expected to confirm that they include the same data!

Reply:
I edited the order of columns being selected for EP_rnaseq_samples and WGS_dneseqsamples to be the same.